euphoria-avro/license-header.txt

```diff
@@ -0,0 +1,13 @@
+Copyright ${license.git.copyrightYears} ${owner}
```

Comment:
This looks like a copy, it should be probably just a symbolic link.

Reply:
hmhm...its copy ...will change